.github/workflows/docker-build-publish.yml

```diff
@@ -20,15 +20,17 @@ jobs:
     permissions:
       contents: write
       packages: write
-    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_pipeline.yml@v0.2.8 # yamllint disable-line rule:line-length
```

Comment:
[question, no change needed] What happened to this comment?

Reply:
I removed the yamllint disable b/c it's not needed. Yamllint doesn't complain on this line.